datahub-gms-graphql-api/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'org.springframework.boot'
+    id 'java'
+}
+
+dependencies {
+    compile project(':datahub-graphql-core')
+    compile (externalDependency.springBootStarterWeb) {
+        exclude module: "spring-boot-starter-tomcat"
+    }
+    compile externalDependency.graphqlJava
+    compile externalDependency.antlr4Runtime
+    compile externalDependency.antlr4
+    compileOnly externalDependency.lombok
+    annotationProcessor externalDependency.lombok
+    compile externalDependency.graphqlJavaSpringWebMvc
```

Comment:
minor organization nitpick: Place these compile statements above the `annotationProcessor` and `compileOnly` statements